ads/pulsepoint.js

```diff
@@ -14,23 +14,17 @@
  * limitations under the License.
  */
 
-import {doubleclick} from '../ads/google/doubleclick';
-import {loadScript, validateData, writeScript} from '../3p/3p';
+import {validateData, writeScript} from '../3p/3p';
 
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function pulsepoint(global, data) {
-  // TODO: check mandatory fields
   validateData(data, [], [
-    'pid', 'tagid', 'tagtype', 'slot', 'timeout',
+    'pid', 'tagid',
```

Comment:
FYI, this is where you specifying optional fields. Just to confirm this is what you want.

Reply:
Thanks for pointing it out, fixed it.